src/components/AppNavigation/CalendarListItemSharing.vue

```diff
@@ -92,8 +93,15 @@ export default {
 			this.isLoading = true
 			this.usersOrGroups = []
 			if (query.length > 0) {
-				const results = await client.principalPropertySearchByDisplayname(query)
-				this.usersOrGroups = results.reduce((list, result) => {
+				
+				const dav = client.principalPropertySearchByDisplayname(query)
+				const circles = this.loadCircles()
+
+				Promise.all([dav, circles]).then(function(results) {
```

Comment:
The function, that you hand over to `Promise.all([...]).then()` gets exactly the same amount as parameters as you hand over in the array to `Promise.all()`.

So in this case you get two parameters: `Promise.all([dav, circles]).then(function(davResults, circlesResults) {` :)